contracts/libraries/PoolSpecs.sol

```diff
@@ -60,6 +63,7 @@ library PoolSpecs {
         uint8 jitThresh_;
         uint8 knockoutBits_;
         uint8 oracleFlags_;
+        bool stableSwap_;
```

Comment:
stableswap pools should be defined based on the poolIdx / pool template used to create the pool so this flag is incorrect

Reply:
poolIdx is not a unique identifier of a pool, two different pools can have same poolIdx

Reply:
poolIdx can define properties of a pool based on the template associated

Reply:
pinging you to discuss this

Reply:
https://github.com/berachain/CrocSwap-protocol/pull/27